threescale_api/resources.py

```diff
@@ -910,8 +910,39 @@ def _extract_resource(self, response, collection) -> Union[List, Dict]:
             extracted = extracted.get(self._entity_collection)
         return extracted
 
+    def select_by(self, **params):
+        """Select by params - logical and
```

Comment:
"Magic" with FILTERS should be also described. Maybe not in docstring certainly in comment at least